core/require.js

```diff
@@ -54,24 +53,22 @@ global.requireHook = function (req) {
         return func(moduleName);
     };
 
-    func.once = function (moduleName, preventRerequire) {
-    	if (preventCache.indexOf(moduleName) !== -1) {
-    		return;
-    	}
-    
+    func.once = function (moduleName) {
         var mod = func(moduleName);
         func.uncache(moduleName);
-        
-        if (preventRerequire) {
-        	preventCache.push(moduleName);
-        }
-        
         return mod;
     };
 
     return func;
 };
 
-hook.setInjectionFunction(function () {
-    require = (global || GLOBAL).requireHook(require);
-});
+module.exports = function () {
+    return {
+        visitor: {
+            Program(path) {
+                path.unshiftContainer('body', babylon.parse('require = (global || GLOBAL).requireHook(require);').program.body[0]);
+            }
+        }
+    };
+};
+exports.default = module.exports;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Newline required at end of file but not found.](https://www.codacy.com/app/theprojectshop/Kassy/file/2651207256/issues/source?bid=3282987&fileBranchId=3324570#l74)